specification/programmableconnectivity/Azure.ProgrammableConnectivity/apis/number.tsp

```diff
@@ -22,12 +22,12 @@ namespace Azure.ProgrammableConnectivity;
   This flow takes place automatically without user interaction, and so only the result
   of the entire flow (the 200 response) is included in this document.
 """)
-interface NumberInterface {
+interface NumberVerification {
```

Comment:
**Thought:** We're suffering a bit here because the CAMARA API is called Number Verification, but it includes both a Verification and a Retrieval endpoint. Maybe it makes sense to call this interface DeviceNumber (by analogy with DeviceLocation), and then we can have DeviceNumberVerificationContent and DeviceNumberVerificaitonResult, etc.

@ellabrmicrosoft This cuts across endpoint naming - what do you think?

Reply:
@AndrewNivenMSFT I'm happy leaving this as is, and adding a lower priority ticket in the backlog to look at the SDK after public preview